src/API/API.js

```diff
@@ -1,41 +1,90 @@
 import React from 'react'
 
 const apiHead = 'https://rancid-tomatillos.herokuapp.com/api/v2'
-const api = {
-  getAllMovies: async () => {
-    const response = await fetch(`${apiHead}/movies`)
-    const data = await response.json()
-    return data.movies
-  },
+class API {
+  // constructor() {
+    // this.apiHead = 'https://rancid-tomatillos.herokuapp.com/api/v2'
+    // can I not use this in static methods because the API wasn't constructed?
```

Comment:
Pretty much! Static methods get called on the class itself, and the constructor doesn't get run unless you instantiate the class itself.